src/components/place-card-list/place-card-list.e2e.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import PlaceCardList from './place-card-list.jsx';
+
+import testData from './place-card-list.test-data.js';
+
+describe(`Main component`, () => {
+// Step 1: Test callback on the rent offer title
+  it(`callback for title is received`, () => {
```

Comment:
it statement должен отвечать на вопрос "что делает". Комментарии при таком подходе просто не нужны
```suggestion
  it(`checks whether callback for title is received`, () => {
```

Reply:
Удалила комментарии из тестов

Reply:
Обновила it statement